modules/swagger-models/src/main/java/io/swagger/models/parameters/AbstractSerializableParameter.java

```diff
@@ -25,7 +26,7 @@
     protected String format;
     protected String collectionFormat;
     protected Property items;
-    protected List<String> _enum;
+    protected List<Object> _enum;
```

Comment:
This change (including getter and setter) is not backward compatible and may affect clients of the `AbstractSerializableParameter` class. I would suggest to apply the same approach that we did for the `default` property (`getDefaultValue` vs `getDefault`, `setDefaultValue` vs `setDefault`)